scvi/distributions/_beta_binomial.py

```diff
@@ -0,0 +1,100 @@
+from typing import Optional
+
+import torch
+from pyro.distributions import BetaBinomial as BetaBinomialDistribution
+from torch.distributions import constraints
+from torch.distributions.utils import broadcast_all
+
+from ._constraints import open_interval, optional_constraint
+
+
+class BetaBinomial(BetaBinomialDistribution):
+    r"""Beta binomial distribution.
+
+    One of the following parameterizations must be provided:
+
+    (1), (`alpha`, `beta`, `total_counts`) where alpha and beta are the shape parameters of
+    the beta distribution and total_counts is the number of trials.
+    (2), (`mu`, `gamma`, `total_counts`), which is the one used by methylVI. These
+    parameters respectively control the mean and dispersion of the distribution.
+
+    In the (`mu`, `gamma`) parameterization, samples from the beta-binomial are generated
+    as follows:
+
+    1. :math:`p_i \sim \textrm{Beta}(\mu, \gamma)`
+    2. :math:`y_i \sim \textrm{Ber}(p_i)`
+    3. :math:`y = \sum_{i}y_i`
+
+    Parameters
+    ----------
+    total_count
+        Number of trials.
+    alpha
+        First shape parameter of the beta distribution.
+    beta
+        Second shape parameter of the beta distribution.
+    mu
+        Mean of the distribution.
+    gamma
+        Dispersion.
+    validate_args
+        Raise ValueError if arguments do not match constraints
+    eps
+        Numerical stability constant
```

Comment:
Probably good to specifically reference how this is used, e.g. "Numerical stability constant used for minimum clamping when the second parameterization is specified."

Reply:
You can also put this under a `Notes` section in the docstring.

Reply:
Added additional details as requested